src/modules/commerce/index.ts

```diff
@@ -386,7 +386,7 @@ export class CommerceModule {
 
     const registrantLength = groupRules.find(
       ([rangeMaximum]) => elementValue <= rangeMaximum
-    )[1];
+    )![1];
```

Comment:
```suggestion
    ) ?? [0, 0])[1];
```

Sadly can't add lines which have not been touched. So I cant set the open bracket in line 387.
Alternitivly we could check if the `find` resolves to `undefined` and throw an error in that case. That would get rid of the non-null assertion.

Reply:
This can never happen. Why write code for it?

Reply:
We will reconsider the implementation in a different PR.